From 9f0e9168b38813e972633b03b5ee6f05d3d03fc8 Mon Sep 17 00:00:00 2001
From: vipinc <vipinc@ami.com>
Date: Wed, 18 Oct 2023 09:41:12 +0530
Subject: [PATCH] Fix build error due to Software.Image yaml

This patch reverts the upstream commit changes causing build error
https://github.com/openbmc/phosphor-dbus-interfaces/commit/8e360784fc4edcc4de5b4b0f1fce5ffa7bf327fb

Tested:
Verified the build is successfull

Signed-off-by: vipinc <vipinc@ami.com>
---
 .../Software/Image/meson.build                | 15 -----------
 gen/xyz/openbmc_project/Software/meson.build  | 15 -----------
 .../Software/Image.errors.yaml                | 10 -------
 .../Software/Image.metadata.yaml              | 27 -------------------
 4 files changed, 67 deletions(-)
 delete mode 100644 gen/xyz/openbmc_project/Software/Image/meson.build
 delete mode 100644 yaml/xyz/openbmc_project/Software/Image.errors.yaml
 delete mode 100644 yaml/xyz/openbmc_project/Software/Image.metadata.yaml

diff --git a/gen/xyz/openbmc_project/Software/Image/meson.build b/gen/xyz/openbmc_project/Software/Image/meson.build
deleted file mode 100644
index f492938..0000000
--- a/gen/xyz/openbmc_project/Software/Image/meson.build
+++ /dev/null
@@ -1,15 +0,0 @@
-# Generated file; do not modify.
-generated_sources += custom_target(
-    'xyz/openbmc_project/Software/Image__cpp'.underscorify(),
-    input: [ '../../../../../yaml/xyz/openbmc_project/Software/Image.errors.yaml',  ],
-    output: [ 'error.cpp', 'error.hpp',  ],
-    depend_files: sdbusplusplus_depfiles,
-    command: [
-        sdbuspp_gen_meson_prog, '--command', 'cpp',
-        '--output', meson.current_build_dir(),
-        '--tool', sdbusplusplus_prog,
-        '--directory', meson.current_source_dir() / '../../../../../yaml',
-        'xyz/openbmc_project/Software/Image',
-    ],
-)
-
diff --git a/gen/xyz/openbmc_project/Software/meson.build b/gen/xyz/openbmc_project/Software/meson.build
index 8608c9f..ace2bcb 100644
--- a/gen/xyz/openbmc_project/Software/meson.build
+++ b/gen/xyz/openbmc_project/Software/meson.build
@@ -89,21 +89,6 @@ generated_others += custom_target(
     ],
 )
 
-subdir('Image')
-generated_others += custom_target(
-    'xyz/openbmc_project/Software/Image__markdown'.underscorify(),
-    input: [ '../../../../yaml/xyz/openbmc_project/Software/Image.errors.yaml',  ],
-    output: [ 'Image.md' ],
-    depend_files: sdbusplusplus_depfiles,
-    command: [
-        sdbuspp_gen_meson_prog, '--command', 'markdown',
-        '--output', meson.current_build_dir(),
-        '--tool', sdbusplusplus_prog,
-        '--directory', meson.current_source_dir() / '../../../../yaml',
-        'xyz/openbmc_project/Software/Image',
-    ],
-)
-
 subdir('RedundancyPriority')
 generated_others += custom_target(
     'xyz/openbmc_project/Software/RedundancyPriority__markdown'.underscorify(),
diff --git a/yaml/xyz/openbmc_project/Software/Image.errors.yaml b/yaml/xyz/openbmc_project/Software/Image.errors.yaml
deleted file mode 100644
index c2fda85..0000000
--- a/yaml/xyz/openbmc_project/Software/Image.errors.yaml
+++ /dev/null
@@ -1,10 +0,0 @@
-- name: UnTarFailure
-  description: An error occurred during untar.
-- name: ManifestFileFailure
-  description: An error when reading the Manifest file.
-- name: InternalFailure
-  description: The operation failed internally during processing the image.
-- name: ImageFailure
-  description: An error occured processing the image.
-- name: BusyFailure
-  description: The device is busy during the update.
diff --git a/yaml/xyz/openbmc_project/Software/Image.metadata.yaml b/yaml/xyz/openbmc_project/Software/Image.metadata.yaml
deleted file mode 100644
index a63d4a1..0000000
--- a/yaml/xyz/openbmc_project/Software/Image.metadata.yaml
+++ /dev/null
@@ -1,27 +0,0 @@
-- name: UnTarFailure
-  level: ERR
-  meta:
-      - str: "PATH=%s"
-        type: string
-- name: ManifestFileFailure
-  level: ERR
-  meta:
-      - str: "PATH=%s"
-        type: string
-- name: InternalFailure
-  level: ERR
-  meta:
-      - str: "FAIL=%s"
-        type: string
-- name: ImageFailure
-  level: ERR
-  meta:
-      - str: "FAIL=%s"
-        type: string
-      - str: "PATH=%s"
-        type: string
-- name: BusyFailure
-  level: ERR
-  meta:
-      - str: "PATH=%s"
-        type: string
-- 
2.25.1

